components/admin_console/server_logs/log_list.jsx

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+const NEXT_BUTTON_TIMEOUT = 500;
+
+export default class Logs extends React.PureComponent {
+    static propTypes = {
+
+        /*
+         * Array of logs to render
+         */
+        logs: PropTypes.arrayOf(PropTypes.string).isRequired,
+        page: PropTypes.number.isRequired,
+        perPage: PropTypes.number.isRequired,
+        nextPage: PropTypes.func.isRequired,
+        previousPage: PropTypes.func.isRequired
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.nextPage = this.nextPage.bind(this);
+        this.previousPage = this.previousPage.bind(this);
+        this.state = {
+            nextDisabled: false
+        };
+    }
+
+    componentDidMount() {
+        this.refs.logPanel.focus();
+    }
+
+    componentDidUpdate() {
+        // Scroll Down to get the latest logs
+        var node = this.refs.logPanel;
+        node.scrollTop = node.scrollHeight;
+        node.focus();
+    }
+
+    nextPage(e) {
+        e.preventDefault();
+
+        this.setState({nextDisabled: true});
+        this.nextTimeoutId = setTimeout(() => this.setState({nextDisabled: false}), NEXT_BUTTON_TIMEOUT);
```

Comment:
Is this to avoid "double clicking" or is there some other purpose to it?

Reply:
As you say, it's just to avoid double clicking.